src/blocks.js

```diff
@@ -13,6 +14,35 @@ export default (editor, opts = {}) => {
     dcb.addDynamciContentBlock();
   }
 
+  //add button block for landing page 
+  const mode = ContentService.getMode(editor);   
+  if (mode === ContentService.modePageHtml) {   
+    bm.add('button', {
+      label: 'Button',
```

Comment:
could be improved by making it translatable

Reply:
Okay @adiux I will make the change as now I know where can I define the translatable labels. Thanks!

Reply:
@adiux I have made necessary changes in the files, need to add labels to the translate En and Fr folder grapesjsbuilder.buttonBlockLabel="Button" for En and grapesjsbuilder.buttonBlockLabel="Bouton" for Fr.